data/hooks/restore/05-conf_ldap

```diff
@@ -40,6 +40,9 @@ else
     mv /etc/ldap/slapd.d "$TMPDIR"
     mkdir -p /etc/ldap/slapd.d
     cp -a "${backup_dir}/ldap.conf" /etc/ldap/ldap.conf
+    cp -a "${backup_dir}/slapd.ldif" /etc/ldap/slapd.ldif
+    # Legacy thing but we need it to force the regen-conf in case of it exist
+    cp -a "${backup_dir}/slapd.conf" /etc/ldap/slapd.conf
```

Comment:
Sounds like tests are passing now :+1: 

But naively this line should fail if the file doesn't exist ? Do you understand why it doesn't ? o.O Or is this script not ran with set -eu ?

Reply:
> Or is this script not ran with set -eu ?

Yes this script don't run set -eu.